x-pack/test/plugin_api_integration/test_suites/task_manager/check_registered_task_types.ts

```diff
@@ -160,6 +160,7 @@ export default function ({ getService }: FtrProviderContext) {
         'osquery:telemetry-saved-queries',
         'report:execute',
         'risk_engine:risk_scoring',
+        'search:agentless-connectors-sync',
```

Comment:
See above comment related to changing "sync" to manager
```suggestion
        'search:agentless-connectors-manager',
```